connectivity/drivers/wifi/COMPONENT_WHD/resources/nvram/TARGET_Cypress/TARGET_CY8CKIT_062_4343W/wifi_nvram_image.h

```diff
@@ -19,7 +19,7 @@
 
 #include <string.h>
 #include <stdint.h>
-#include "generated_mac_address.txt"
+#include "../generated_mac_address.txt"
```

Comment:
Don't know, it would be for sure the best solution, but I don't know how to add it in the include paths. Another option is to move the file inside the target folder, but as they are all the same i've done in this way adding ../

Reply:
the old tools should have the include as its in TARGET_Cypress. Is this an issue with CMake only ?

Reply:
Please do not edit or delete the the wifi_nvram_image.h files, or any file under COMPONENT_WHD. Those files come from our Wi-Fi Host Driver library as is. Modifications to them will not be maintained in future releases of that library.

Instead, make sure that the include paths are correct such that the mac address header is included in the build without modifications to the header.